stdlib/src/builtin/object.mojo

```diff
@@ -293,52 +213,22 @@ struct _ObjectImpl(
     """Type discriminator indicating a string."""
     alias list: Int = 5
     """Type discriminator indicating a list."""
-    alias dict: Int = 8  # TODO
+    alias dict: Int = 8
     """Type discriminator indicating a dictionary."""
     alias function: Int = 6
     """Type discriminator indicating a function."""
     alias obj: Int = 7
     """Type discriminator indicating an object."""
+    alias tuple: Int = 8
```

Comment:
**Question** Why is it allowed to have `dict` and `tuple` to have the same value for the type discriminator?  Is that intended?

Reply:
Nope, it's fixed with next commit :+1: 
(we should also be able to get this from variant and simplify by following the variant.Ts order)